examples/src/main/java/tachyon/examples/Performance.java

```diff
@@ -228,18 +227,17 @@ public TachyonReadWorker(int id, int left, int right, ByteBuffer buf) throws IOE
     }
 
     public void readPartition() throws IOException {
-      TachyonByteBuffer buf;
       if (sDebugMode) {
+        ByteBuffer buf = ByteBuffer.allocateDirect((int) sBlockSizeBytes);
```

Comment:
DirectBuffer can be optimal for I/O, since this is performance example. There are other places in this file where `allocateDirect` are used.